guarddog/analyzer/sourcecode/clipboard-paste.yml

```
@@ -0,0 +1,10 @@
+rules:
+  - id: clipboard-paste
```

```suggestion
  - id: clipboard-access
```

so we could also cover stealing data from the clipboard in the future?